src/app/modules/OpenshiftStreams/dialogs/CreateInstance/api.ts

```diff
@@ -67,22 +67,26 @@ export const useAvailableProvidersAndDefault = () => {
       undefined,
       InstanceType.eval
     );
+
     return (res.data.items || [])
       .filter(
         (p) =>
-          p.enabled &&
-          p.capacity.some(
-            (c) =>
-              c.instance_type === instance_type &&
-              c.max_capacity_reached === false
-          )
+          p.enabled && p.capacity.some((c) => c.instance_type === instance_type)
       )
       .map((r): RegionInfo => {
+        const max_capacity_reached = r.capacity?.some(
+          (c) =>
+            c.max_capacity_reached === true && c.instance_type === instance_type
+        );
         return {
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           id: r.id!,
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          displayName: r.display_name!,
+          displayName: `${r.display_name!}${
+            max_capacity_reached ? ' (not supported)' : ''
```

Comment:
I think we should add this bit into the UI component, so to have it under i18n

Reply:
@jgiardino make sense. I'll remove this extra string in this PR.